x/dualstaking/keeper/delegator_reward.go

```diff
@@ -110,6 +119,14 @@ func (k Keeper) CalcEffectiveDelegationsAndStake(stakeEntry epochstoragetypes.St
 // CalcDelegatorReward calculates a single delegator reward according to its delegation
 // delegatorReward = delegatorsReward * (delegatorStake / totalDelegations) = (delegatorsReward * delegatorStake) / totalDelegations
 func (k Keeper) CalcDelegatorReward(delegatorsReward math.Int, totalDelegations math.Int, delegation types.Delegation) math.Int {
+	// Sanity check - totalDelegations > 0
+	if totalDelegations.IsZero() {
+		utils.LavaFormatWarning("totalDelegations is zero", fmt.Errorf("critical: Attempt to divide by zero"),
```

Comment:
total delegations can be zero, dont print a warning here

Reply:
Like this? cfe747751c76af0a4a8e2e9f66ce55efd7d6bb46